layouts/shortcodes/protobuf.html

```diff
@@ -0,0 +1,3 @@
+{{ $protoName := .Get "name" }}
+{{ $protoDocsUrl := index .Site.Data.ProtoMap.Rel $protoName }}
+<a href="{{ .Site.BaseURL }}{{ $protoDocsUrl }}">{{ $protoName }}</a>
```

Comment:
doesn't this also need to be versioned? or is the version implicit in the `protoDocsUrl`? if so, can we comment that?

Reply:
typically our version has gone before `{{ .Site.BaseURL }}`, so seems like it's missing

Reply:
good point, that needs a version as well. It's not currently used but will be in a follow up, I will fix this here though